deploy/chart/templates/0000_90_olm_01-prometheus-rule.yaml

```diff
@@ -9,12 +9,23 @@ metadata:
     role: alert-rules
 spec:
   groups:
-    - name: olm.failing_operators.rules
+    - name: olm.csv_abnormal.rules
       rules:
-        - alert: FailingOperator
+        - alert: CsvAbnormalReplacingOver30Min
+          expr: csv_abnormal{phase=~"(^Replacing$|^Failed$|^Unknown$)",exported_namespace=~"(^openshift.*|^kube.*|^redhat.*|^default$)"}
+          for: 30m
+          labels:
+            severity: warning
+            namespace: {{ printf "{{ $labels.namespace }}" }}
```

Comment:
OLM uses helm3 in CI to generate manifests. This is not very friendly for Go template which is used in alerts... 

We render the namespace here using `{{ printf "{{ $labels.namespace }}" }}`, but helm3 failed to render it and I got error `Error: YAML parse error on olm/templates/0000_90_olm_01-prometheus-rule.yaml: error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{"$labels.namespace":interface {}(nil)}`.

I cannot figure out a way to render this template with helm3, any ideas @RiRa12621? Maybe we can drop this label if it is not very important.

Reply:
I can add this `namespace` label to the annotation message if it is not really necessary to keep this label.

Reply:
in theory i'd expect that label to exist without explicitly defining it, can you test that?
We depend on that label being present for our alert routing

Reply:
Let me try it.